src/pages/api/feedback/index.page.ts

```diff
@@ -1,15 +1,44 @@
 import { NextApiRequest, NextApiResponse } from 'next';
+import prisma from '../../../api/prisma/prisma';
 
 async function handler(req: NextApiRequest, res: NextApiResponse) {
-  const feedback = req.body;
-  fetch(process.env.SLACK_FEEDBACK_WEBHOOK, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify({ text: feedback?.text }),
-  });
-  res.send(200);
+  const { text, email, route } = req.body;
+
+  if (typeof text !== 'string' || text.length === 0) {
+    return res.status(400).send('Feedback text is required');
+  }
+
+  try {
+    // We'd like to record the full feedback entry in our database for future reference.
+    await prisma.feedback.create({
+      data: {
+        email: email ?? 'No email provided',
```

Comment:
can we make this column nullable instead?

Reply:
nice catch, this is nullable by default so I didn't really need to have this text fallback. if `email` is undefined/null this field will be null too